wallet/src/wallet/changeset.rs

```diff
@@ -268,6 +318,35 @@ impl ChangeSet {
             })?;
         }
 
+        // Insert locked outpoints.
+        let mut stmt = db_tx.prepare_cached(&format!(
+            "INSERT INTO {}(txid, vout, is_locked) VALUES(:txid, :vout, :is_locked) ON CONFLICT DO UPDATE SET is_locked=:is_locked",
+            Self::WALLET_OUTPOINT_LOCK_TABLE_NAME,
+        ))?;
+        let locked_outpoints = &self.locked_outpoints.locked_outpoints;
+        for (&outpoint, is_locked) in locked_outpoints.iter() {
+            let OutPoint { txid, vout } = outpoint;
+            stmt.execute(named_params! {
```

Comment:
Also if we remove the `is_locked` column then you'd need the insert_stmt and a delete_stmt to remove outpoints where the changes has `is_locked=true`.